fendermint/vm/iroh_resolver/src/iroh.rs

```diff
@@ -135,9 +127,9 @@ fn start_resolve<V>(
                             });
                         }
                     }
-                    Ok(Err(e)) => {
+                    Err(e) | Ok(Err(e)) => {
```

Comment:
vote "nay" in the case of any error

Reply:
It would mean that if the service on the other side is not responding, we will re-enqueue to task, right? does that mean if the other service is not responding then the task queue will keep growing?

Reply:
Ah Okay. sounds good.